engine/apps/schedules/ical_utils.py

```diff
@@ -589,6 +589,8 @@ def create_base_icalendar(name: str) -> Calendar:
     cal.add("x-wr-calname", name)
     cal.add("x-wr-timezone", "UTC")
     cal.add("prodid", "//Grafana Labs//Grafana On-Call//")
+    # suggested minimum interval for polling for changes
+    cal.add("REFRESH-INTERVAL;VALUE=DURATION", "P1H")
```

Comment:
Does it mean that Google calendar will refresh an exported ICal file every hour **at most**?
If that's the case, is it possible to set a lower value to get it update faster?

Reply:
Not really, this is just a "suggestion" on how much (minimum) time wait before trying a refresh... Google is expected to still sync every 12-24hs. I didn't want to set a too lower threshold just in case some other clients keep hitting the endpoint (and 1h seemed frequent enough?).